### PR TITLE
FIX: Remove ui lib

### PR DESCRIPTION
**Pull request by cristian-aldea**
This pr removes the ui-lib from our project

Notes:
* fixed a failing test in mull-api
* fixed linting issues

### CODE REVIEW

File: apps/mull-api/src/app/auth/auth.service.spec.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const mockUserService = () => ({
         return [cloneDeep(mUser)];
       }
     }
-    return null;
+    return [];
```
Comment (JimLau49) [inline apps/mull-api/src/app/auth/auth.service.spec.ts:17]: Nice to see this fixes travis now 🚀

**Review by JimLau49 — DISMISSED** [review]
Alright looks good to me, thanks for refacoring out ui-lib

File: apps/mull-ui/src/app/app.tsx [hunk 2]
```diff
@@ -50,7 +50,6 @@ export const App = () => {
               <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
               <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
               <Route path={ROUTES.MY_EVENTS} component={() => <div>MY_EVENTS!</div>} />
-              <Route path={ROUTES.HOME} render={() => <Redirect to={ROUTES.DISCOVER} />} />
```
Comment (cristian-aldea) [inline apps/mull-ui/src/app/app.tsx]: This was a suggestion I gave to Ragith to redirect to `/home/discover` if the user goes to `/home/somethingelse`, however, this seems to break the SwipeableRoutes slightly, so I decided to remove it for now

**Review by r-saba — COMMENTED** [review]
Everything seems to be working and looking good! I see some linting warning for mull-api about unused variables, can we fix those before merging?

File: apps/mull-ui/src/app/app.tsx (hunk 2 above)
Comment (r-saba) [reply apps/mull-ui/src/app/app.tsx]: Feel free to make a bug report about `/home/somepage` I can fix it

**Comment by cristian-aldea** [comment]
> Everything seems to be working and looking good! I see some linting warning for mull-api about unused variables, can we fix those before merging?

Sure I can rework that code. I didn't fix that issue because I didn't think it was important, but you're right might as well fix everything.

File: apps/mull-ui/src/app/app.tsx (hunk 2 above)
Comment (cristian-aldea) [reply apps/mull-ui/src/app/app.tsx]: Ok ill remove it from this pr

**Comment by cristian-aldea** [comment]
I fixed the linting issues in mull-api. There is one more lint issue in mull-ui, but that will be fixed anyways as the code is placeholder code.

There is a failing test, but that is not due to my changes, the test itself is flaky, and can fail depending on the date of month it is run on. If we run the test on the last day of a month, it fails. I'll make a bug ticket for it: #107

**Review by r-saba — APPROVED** [review]
I see the bug reports have been made. Good work and thanks for cleaning up the project, LGTM!

**Review by JimLau49 — APPROVED** [review]
Took a second look still LGTM! good work 🚀

**Pull request merged**